<launch>
	<arg name="port" default="/dev/ttyACM0" doc="Port used by the Myo dongle. Check it with 'ls /dev/' and look for ttyACM0 for instance. You need to provide the full path." />
	<node pkg="ros_myo" type="myo-rawNode.py" name="myo_raw" output="log" args="$(arg port)" />
</launch>
